go/vt/vttablet/tabletserver/tabletserver_test.go

```diff
@@ -640,53 +633,6 @@ func TestTabletServerStopWithPrepare(t *testing.T) {
 	}
 }
 
-func TestTabletServerMasterToReplica(t *testing.T) {
```

Comment:
Why did you delete these tests (this and the ReplicaToMaster)? Are they failing? If so, we need to investigate why.

These are important because they prove that no errors will be served during state transitions.

Reply:
They were removed by mistake. Sorry about that, and thanks for catching it!

The tests are related to the [5a449d1](https://github.com/vitessio/vitess/pull/4310/commits/5a449d1597509a85412a8ae84a8b931c6d75321b) commit. I don't think that change is correct, and the tests agree :)